torch/optim/_muon.py

```diff
@@ -61,7 +66,7 @@ def zeropower_via_newtonschulz(G: Tensor, ns_config: BaseMsignFnConfig) -> Tenso
     if G.size(0) > G.size(1):
         X = X.T
     # Ensure spectral norm is at most 1
-    X = X / (X.norm() + 1e-7)
+    X = X / (X.norm() + EPS)
```

Comment:
might also be good to add it explicitly to function arguments: like `..., eps = EPS):` ? (and maybe same with coefficients?) this would allow someone to call the function with other eps without modifying the global vars

Reply:
good idea, I added eps to `NewtonSchulzConfig`.